sources/schema/club.js

```diff
@@ -54,12 +59,12 @@ const queries = `
     ):[Club]
     clubByName(name:String):Club
     clubById(id:String):Club
+    deleteClub(id:ID!):ResponseResult
 `;
 
 const mutations = `
-    addClub(club:ClubInputType):Club,
-    updateClub(id:ID,club:ClubInputType):Club,
-    deleteClub(id:ID):Club,
+    addClub(club:ClubInputType):ClubResult,
```

Comment:
Mutation should return Responseresult?

Reply:
yes yes, thanks for pointing it out.

Reply:
so there are two kinds of mutations,
1. which returns a result object
2. which just returns a success message

addClub returns the created club object, but mutations like deleteClub only returns a success message.